zetaclient/zetabridge/zetacore_bridge.go

```diff
@@ -258,7 +258,12 @@ func (b *ZetaCoreBridge) UpdateZetaCoreContext(coreContext *corecontext.ZetaCore
 		tssPubKey = tss.GetTssPubkey()
 	}
 
-	coreContext.Update(keyGen, newChains, newEVMParams, newBTCParams, tssPubKey, init, b.logger)
+	crosschainFlags, err := b.GetCrosschainFlags()
```

Comment:
The value is still used if `err != nil`, what is it?
We should use a fallback value where everything is disabled in the flags.
We should define it here if `err != nil` as we should never use a returned value when a function return an error by convention

Reply:
The value is an empty struct, it is by default disabling inbound and outbound. I can make the value more clear here.

Reply:
Seems to me there are three errors need to be handled here:

1. `keyGen, err := b.GetKeyGen()}`
2. `tss, err := b.GetCurrentTss()`
3. `crosschainFlags, err := b.GetCrosschainFlags()`

A easy way I can think of to avoid using wrong data (when err occurs) is to make a `return` on each error if the function `UpdateZetaCoreContext` can't make all queries successfully in one shot (partially update core context fields could lead to unknown behaviors), similar to what we did for `chains, err := b.GetSupportedChains()`. Ideally we also print a log (so we can see in Datadog) with `Info` level at least. This one-shot model would make method `coreContext.Update`'s logic easier.